apl/scalar-functions/string-functions.mdx

```diff
@@ -721,10 +721,10 @@ Backreferences match the same text as previously matched by a capturing group. W
 
 ```kusto
 ['github-issues-event']
-| project backreferences = replace_regex(@'observability=(\d+)', 'axiom=$1', creator)
+| project backreferences = replace_regex(@'observability=(d+)', 'axiom=$1', creator)
```

Comment:
that's not matching a number though now

Reply:
@Rambatino Oh, how dumb of me. I've replaced `d` with `.` so that it makes sense but avoids the `\`. Please let me know if this works.

Reply:
yeah seems good!